ports/espressif/boards/espressif_esp32s3_lcd_ev_v1.5/sdkconfig

```diff
@@ -0,0 +1,28 @@
+#
+# Espressif IoT Development Framework Configuration
+#
+#
+# Component config
+#
+#
+# ESP System Settings
+#
+CONFIG_ESP_CONSOLE_UART_CUSTOM=y
+# CONFIG_ESP_CONSOLE_NONE is not set
+CONFIG_ESP_CONSOLE_UART=y
+CONFIG_ESP_CONSOLE_UART_CUSTOM_NUM_0=y
+CONFIG_ESP_CONSOLE_UART_NUM=0
+CONFIG_ESP_CONSOLE_UART_TX_GPIO=43
+CONFIG_ESP_CONSOLE_UART_RX_GPIO=44
+CONFIG_ESP_CONSOLE_UART_BAUDRATE=115200
+# end of ESP System Settings
+
+#
+# LWIP
+#
+CONFIG_LWIP_LOCAL_HOSTNAME="circuitpy"
+# end of LWIP
```

Comment:
You shouldn't need any of this.

Reply:
Sorry, I meant the file should be empty. The build fails with it missing.

Reply:
Thank you, I am a newbie for build the Circuitpython.